src/perf.cc

```diff
@@ -91,8 +91,8 @@ perf_event_attr perf_config_from_watcher(const PerfWatcher *watcher,
   attr.sample_stack_user = watcher->sample_stack_size;
 
   // If use_kernel is requested false --> exclude_kernel == true
-  if (watcher->options.use_kernel == PerfWatcherUseKernel::kTry)
-    attr.exclude_kernel = true;
+  if (watcher->options.use_kernel != PerfWatcherUseKernel::kOff)
+    attr.exclude_kernel = false;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏ 
```suggestion
   attr.exclude_kernel = (watcher->options.use_kernel == PerfWatcherUseKernel::kOff);
```